nevergrad/parametrization/data.py

```diff
@@ -430,6 +432,16 @@ class Array(Data):
 
     value: core.ValueProperty[tp.ArrayLike, np.ndarray] = core.ValueProperty()
 
+    def smooth_copy(self: A, possible_radii: tp.List[int] = None) -> A:
+        candidate = self.copy()
```

Comment:
please make it a function near the optimizer, not in the parametrization,
+ use spawn_child instead of copy
+ don't use np.random, use the parameter.random_state instead

Reply:
Done.